### DISCUSSION TITLE
Kötegelten könyvelt bizonylatok analitikája

### DISCUSSION THREAD

**Discussion by migpeti** [nav-gov-hu/eVAT#24] [Kérdések és válaszok - Gépi interfész] [answered]
Üdvözlöm!

Sok helyen bevett gyakorlat a bizonylatok kötegelt könyvelése. Gondolok itt jellemzően az egy napon kiállított nyugták, pénztárgép slip alapján történő könyvelésére. Továbbá akár a készpénzes számlák, összesítő alapján történő könyvelésére. Ezekben az esetekben a számla analitika csak a konkrét értékesítő rendszerben érhető el. A könyvelési rendszerben csak a kapott összesítők alapján elkészített vegyes feladás szerepel. Ezek a vegyes feladások vajon maradhatnak-e a jövőben is (mondjuk sourceDocumentType=OTHER,  és sourceDocumentId=SajátAzonosító)? Vagyis lehet-e csak azok adatait szerepeltetni, vagy szükséges kifejleszteni egy mechanizmust, hogy a könyvelt vegyes bizonylatok "mögötti" bizonylatok listája tételesen belekerüljön az analitika XML-be?

Sőt kicsit tovább feszítve a húrt: Egy ellenőrzés során vajon gondot jelent-e ha a könyvelési rendszerben az előbbiekben említett vegyes bizonylat szerepel,é s a Data XML-ben beküldött analitika továbbra is csak az adott értékesítő szakrendszerből érhető el.

Előre is köszönöm a válaszukat.
Péter

**Comment by NTCA-developer** [comment, marked as answer]
Kedves @migpeti !

A szakmai részt majd megválaszolják az illetékesek, de azt elárulom hogy a sourceDocumentType kizárólag azért került a sémába, mert lesznek olyan cégek akiknek az lesz a fő use case hogy letöltik tőlünk az analitikájukat, és ennek során segíteni szerettük volna őket hogy az egyes analitika tételek mellé nem csak odahajítjuk a bizonylatszámot hogy "asasdasf/2023" hanem odatesszük hogy milyen forrásból jött, ha egyszer már úgyis tudjuk a mi oldalunkon és tételesen megvan. Ennek okán a sourceDocumentType opcionális lett, igazából az sem baj ha nem adják meg amikor feltöltik az analitikát. Az OTHER tényleg az egyéb bizonylatok miatt van a sémában, pl egy felvásárlási okirat, ezért szerintem nem szerencsés erőltetni akkor ha egyébként nem muszáj.

Tudom ez csak részben válasz az  eredeti kérdésre, de gondoltam megosztom az infót.

**Comment by NTCA-HM** [comment]
Kedves @migpeti !

A korábbi válasz kiegészítése érdekében: Ez nem okoz problémát, ahogy leírásra került az megfelelő. Számviteli politika kérdése és ott ezt a jellegű működést szerepeltetni is kell.
